Fix running unit tests on the CI

# Summary
Some of the features for the database and the REST API require accessing files on disk, which we do not have access to on Github. There are probably some way to get around this or mock accessing the disks. However, it is not  a very high priority right now provided that developers run the test suite on their own first. 

# Example
An example of the way that it is currently broken is this run of the CI during the development of the database: https://github.com/baas-project/baas/runs/4091566066?check_suite_focus=true

Hi, @ValentijnvdBeek
I resolved privileged problem to access some files. 
but it cannot to be passed the test of config_test.go because I cannot find the example of `/etc/baas.toml`.
Could tell me how to write that config file?

https://github.com/u5surf/baas/runs/4196057947?check_suite_focus=true
```
[2021-11-13 00:59:50] - ERROR - Cannot open the configuration file: 'open /etc/baas.toml: no such file or directory'
Failed to fire hook: sending log: Post "http://control_server:4848/log": dial tcp: lookup control_server: Temporary failure in name resolution
[2021-11-13 00:59:50] - ERROR - Cannot read configuration file: 'invalid argument'
Failed to fire hook: sending log: Post "http://control_server:4848/log": dial tcp: lookup control_server: Temporary failure in name resolution
[2021-11-13 00:59:51] - INFO - Creating the configuration file object
--- FAIL: TestConfigFile (0.10s)
    config_test.go:11: 
        	Error Trace:	config_test.go:11
        	Error:      	Should be true
        	Test:       	TestConfigFile
    config_test.go:12: 
        	Error Trace:	config_test.go:12
        	Error:      	Should be true
        	Test:       	TestConfigFile
FAIL
FAIL	github.com/baas-project/baas/management_os/entrypoint	0.108s
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -1,22 +1,19 @@
 on: [push]
 name: CI
 jobs:
-# CI testing does not work at the moment since we need to write to the disk, but we do not
-# have the permission to access it. I don't feel like fixing it right now. Please tag me
-# whenever you do end up solving this issue so that I know how many years have passed.
-# Until that time, please manually run the test suite before pushing. Thanks. 
-#   - ValentijnvdBeek
-#  test:
-#    runs-on: ubuntu-latest
-#    steps:
-#      - name: Install Go
-#        uses: actions/setup-go@v2
-#        with:
-#          go-version: 1.16
-#      - name: Checkout code
-#        uses: actions/checkout@v2
-#      - name: Build
-#        run: go test ./...
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build
+        run: |
+               sudo cp management_os/config/config.toml /etc/baas.toml
+               sudo /opt/hostedtoolcache/go/1.16.10/x64/bin/go test ./...
   lint:
     runs-on: ubuntu-latest
     steps:

--- a/AUTHORS
+++ b/AUTHORS
@@ -1,0 +1,7 @@
+# This is the list of baas-project/baas's authors for copyright purposes.
+# This does not necessarily list everyone who has contributed code, since
+# in some cases,
+# their employer may be the copyright holder. To see the full list of
+# contributors, see the revision history in source control.
+
+u5surf <u5.horie@gmail.com>

--- a/management_os/config/config.toml
+++ b/management_os/config/config.toml
@@ -1,3 +1,3 @@
-uploadDisk = false
+uploadDisk = true
 rebootAfterFinish = false
 setNextBoot = true

